### PR TITLE
#3302: Absolute basic UT

### PR DESCRIPTION
**Pull request by Xivolkar**
tests: Adding Unit tests

Some work on adding unit tests for `probe/cri`. This PR adds absolute basic unit tests.

Partially fixes https://github.com/weaveworks/scope/issues/3302

P.S: I'm a beginner to Go. Open to listening for things I can do better.

### CODE REVIEW

File: probe/cri/registry_test.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import (
+	"testing"
+
+	"github.com/weaveworks/scope/probe/cri"
+)
+
+func TestParseHttpEndpointUrl(t *testing.T) {
+	_, err := cri.NewCRIClient("http://xyz.com")
+
+	if err == nil {
```
Comment (bboreham) [inline probe/cri/registry_test.go]: This is fine, however you might want to look at `assert.Error` from the package `github.com/stretchr/testify/assert`.
We use this package in a few places, e.g. https://github.com/weaveworks/scope/blob/aaa63bbed0c91945477780b96d0be02c7cb9c3c6/prog/main_test.go#L22

It just neatens things up a bit compared to the three-line if-statement.

Comment (Xivolkar) [reply probe/cri/registry_test.go]: Sure, that sounds good. I'll do it

**Comment by bboreham** [comment]
Thanks!  I've kicked off full integration tests.

**Comment by Xivolkar** [comment]
@bboreham checks passed.

**Pull request merged**